app/db/AtomWorkshopDB.scala

```diff
@@ -55,5 +58,19 @@ object AtomWorkshopDB {
     transformAtomLibResult(datastore.getAtom(AtomWorkshopDB.buildKey(atomType, id)))
   }
 
+  def updateAtom(datastore: DynamoDataStore[_ >: ExplainerAtom with CTAAtom with MediaAtom], atomType: AtomType, id: String, field: String, value: JsLookupResult) = {
```

Comment:
Good point. We need to document this I suspect, unless we can find a way to represent the type more nicely.